pkg/machine/wsl/usermodenet.go

```diff
@@ -29,6 +29,8 @@ fi
 if [[ ! $ROUTE =~ default\ via ]]; then
 	exit 3
 fi
+# auto-correct previous installs of known bad image v39.0.3
+test -x /usr/local/bin/vm || chmod 755 /usr/local/bin/vm
```

Comment:
any reason to do the test vs just always the chmod?

Reply:
no real convincing reason, other than to save an exec since test is a builtin, but that won't really make any discernible difference, so I just changed it.